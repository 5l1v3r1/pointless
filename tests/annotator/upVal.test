
foo(l) = (m, n) => l