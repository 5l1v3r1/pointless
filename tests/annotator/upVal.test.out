
foo(l) = (m, n) => l
access: l local: 2
entry: m index: 0
entry: n index: 1
entry: l index: 2 upIndex: 0
entry: l index: 0
entry: foo index: 0
